sdk/core/Azure.Core/src/AzureSasCredential.cs

```diff
@@ -25,6 +29,23 @@ public string Signature
             private set => Volatile.Write(ref _signature, value);
         }
 
+        /// <summary>
+        /// The ticks value of the DateTimeOffset when the shared access signature was last set.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public long SignatureUpdated
```

Comment:
Use DateTimeOffset?

Reply:
Volatile doesn't work with DateTimeOffset.

Reply:
Could do a Volatile.Read on the Ticks property, but I guess it doesn't matter much since this is hidden anyways.